bundles/org.openhab.core.io.console.karaf/src/main/java/org/openhab/core/io/console/karaf/internal/CompleterWrapper.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.core.io.console.karaf.internal;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.apache.karaf.shell.api.console.Candidate;
+import org.apache.karaf.shell.api.console.CommandLine;
+import org.apache.karaf.shell.api.console.Session;
+import org.apache.karaf.shell.support.completers.StringsCompleter;
+import org.eclipse.jdt.annotation.NonNull;
+import org.openhab.core.io.console.Completer;
+import org.openhab.core.io.console.extensions.ConsoleCommandExtension;
+
+/**
+ *
+ * @author Cody Cutrer - Initial contribution
+ */
+public class CompleterWrapper implements org.apache.karaf.shell.api.console.Completer {
```

Comment:
Is it possible to annotate the class with `@NonNullByDefault`? Also please add a class description in JavaDoc

Reply:
I was having trouble with that. The interface is not `@NonNullByDefault`, and I couldn't figure out the proper encantation of `@Nullable` on the overridden methods to make it compatible. And given that the only methods in this are overridden from the interface, I didn't think it was worth it.

Reply:
Ok. Probably `@NonNullByDefault({})` would have done the trick.